### PR TITLE
Merge cran_upload to develop and main branches

### PR DESCRIPTION
**Pull request by iskandari**

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -18,3 +18,5 @@ cran-comments.md
 ^README\.Rmd$
 NOTICE
 ^LICENSE\.md$
+^src/includes/libiris2odim$
```
Comment (adokter) [inline .Rbuildignore:21]: iris has to stay available on main

File: .gitignore [hunk 2]
```diff
@@ -5,8 +5,14 @@
 # Session Data files
 .RData
 
+#VS-code specific files
+.vscode/
+
 output.csv
 
+# Auto-generated vscode files
+.vscode/
```
Comment (adokter) [inline .gitignore:14]: duplicate vscode

File: inst/COPYRIGHTS [hunk 3]
```diff
@@ -56,54 +56,6 @@ along with RAVE.  If not, see <http://www.gnu.org/licenses/>.
 
 ====
 
-libiris2odim is written Mark Couture, Environment Canada, with contributions
```
Comment (adokter) [inline inst/COPYRIGHTS:59]: these copyrights should be kept on the main branch

**Review by adokter — CHANGES_REQUESTED** [review]

**Comment by iskandari** [comment]
Ideas on how to resolve the warning message in Windows build @adokter @andershenja ?  

```
❯ checking compiled code ... WARNING
  File 'vol2birdR/libs/x64/vol2birdR.dll':
    Found 'exit', possibly from 'exit' (C), 'stop' (Fortran)
      Object: './libiris2odim/iris2odim.o'
  
  Compiled code should not call entry points which might terminate R nor
  write to stdout/stderr instead of to the console, nor use Fortran I/O
  nor system RNGs nor [v]sprintf.
  ```

Seems like direct `exit()` calls have already been replaced:
https://github.com/adokter/vol2birdR/blob/7325c1a80986b0f9c058046bd6ff9b682522032d/src/libiris2odim/iris2odim.c#L62-L78

File: inst/COPYRIGHTS (hunk 3 above)
Comment (iskandari) [reply inst/COPYRIGHTS:59]: reinserted

**Review by adokter — APPROVED** [review]

**Pull request merged**